templates/patterns/card_layout/pattern-card-layout.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Card layout pattern.
+ */
+#}
+{% set _items = [] %}
+{% for _item in items %}
+  {% set _item = _item|merge({
```

Comment:
We need to add all the same tags that we have in the card one, so that the same result comes out. So m2 margin, div, etc.